src/test/java/uk/gov/hmcts/divorce/citizen/notification/ApplicationOutstandingActionNotificationTest.java

```diff
@@ -81,10 +84,35 @@ void shouldCallSendEmailForSupportingDocuments() {
         verify(commonContent).templateVarsForApplicant(data, data.getApplicant1(), data.getApplicant2());
     }
 
+    @Test
+    @SuppressWarnings("squid:S6068")
```

Comment:
what is this suppression for ?

Reply:
not sure, I'll remove it. Just copied it in as every other test in the file has it